nativeruntime/cpp/libcutils/ashmem.cpp

```diff
@@ -63,8 +63,13 @@ int ashmem_valid(int fd) {
 
 int ashmem_create_region(const char* /*ignored*/, size_t size) {
   char pattern[PATH_MAX];
+  #if defined(_WIN32)
```

Comment:
Currently, on Windows, if CursorWindows are going to be in-memory only, I do not think we need to create tmp files, as they will be unused.

Reply:
I will move Windows' implementation to in-memory only at next commit. Thanks.

Reply:
I disabled `robo_android_database_SQLiteConnection#createAshmemRegionWithData` for Windows at new commit.